amp.php

```diff
@@ -14,13 +14,13 @@
  */
 
 define( 'AMP__FILE__', __FILE__ );
-define( 'AMP__DIR__', __DIR__ );
+define( 'AMP__DIR__', __DIR__ ); // phpcs:ignore PHPCompatibility.Keywords.NewKeywords.t_dirFound
```

Comment:
Maybe this should rather go back to being:

```suggestion
define( 'AMP__DIR__', dirname( __FILE__ ) );
```

This would prevent a PHP warning in PHP 5.2 from being raised. But that would be the only benefit to doing so. The plugin would be short-circuting before this constant is actually used to include a file.

Reply:
I'd probably avoid throwing a notice on PHP 5.2 in this particular case, but I don't feel strongly about it.